File: java-project/BankingSystem.java
// BankingSystem.java

// Placeholder code for Simple Banking System
public class BankingSystem {
    static class BankAccount {
        String accountHolder;
        double balance;

        public BankAccount(String accountHolder, double balance) {
            this.accountHolder = accountHolder;
            this.balance = balance;
        }

        public void deposit(double amount) {
            balance += amount;
            System.out.println(amount + " deposited. Current balance: " + balance);
        }

        public void withdraw(double amount) {
            if (balance >= amount) {
                balance -= amount;
                System.out.println(amount + " withdrawn. Current balance: " + balance);
            } else {
                System.out.println("Insufficient funds");
            }
        }

        public void getBalance() {
            System.out.println("Current balance: " + balance);
        }
    }

    public static void main(String[] args) {
        System.out.println("Welcome to the Simple Banking System!");
        BankAccount account = new BankAccount("Akshit", 1000);
        account.deposit(500);
        account.withdraw(200);
        account.getBalance();
    }
}

File: python-project/banking_system.py
# banking_system.py

# Placeholder code for Simple Banking System
class BankAccount:
    def __init__(self, account_holder, balance=0):
        self.account_holder = account_holder
        self.balance = balance

    def deposit(self, amount):
        self.balance += amount
        print(f"{amount} deposited. Current balance: {self.balance}")

    def withdraw(self, amount):
        if self.balance >= amount:
            self.balance -= amount
            print(f"{amount} withdrawn. Current balance: {self.balance}")
        else:
            print("Insufficient funds")

    def get_balance(self):
        print(f"Current balance: {self.balance}")

# Placeholder main function
def main():
    print("Welcome to the Simple Banking System!")
    account = BankAccount("Akshit", 1000)
    account.deposit(500)
    account.withdraw(200)
    account.get_balance()

if __name__ == "__main__":
    main()
